content/developer/embedding/index.md

```diff
@@ -3,46 +3,125 @@ title: Embedding
 tags:
 - Features
 - Embedding
-category: help
+category: developer
 menu:
   developer:
-    identifier: embedding
+    identifier: embedding-developer
 aliases:
-- /features/embedding
-- /help/forum-settings/embedding
+- /features/embedding/advanced
+- /developer/embedding/advanced
 ---
 
-## Embedding Forum
+## Overview
 
-Embedding Vanilla Forums works by putting a snippet of code on your website page, and may be a choice if you want to insert your forum into a site quickly. The snippet points to some javascript and the forum is embedded into the page using an iframe
```

Comment:
Looks like you've replaced the contents of the embedding page with the advanced page. It should be appended, not deleted.

Reply:
Handled and by todd's request, the combined page is in the help section